product_docs/docs/pgd/5/raft/03_migrating_to_raft_subgroups.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Migrating to Raft subgroups
+---
+
+This section assumes that you have an already running cluster and wish to introduce Raft subgroups to your cluster.
+
+
+## Migrating to Raft subgroups (without TPA)
+
+A number of configuration steps need to be taken to enable Raft subgroups to an existing cluster.
+
+* A “top-level” group needs to be defined for all nodes in the PGD cluster. Use bdr.create_node_group
+* A subgroup needs to be created for each location. Use bdr_create_node_group
+* Each location’s subgroup should be added to the “top-level” group. Use bdr.join_node_group(), bdr.switch_node_group() or bdr.alter_node_group().
+* Each node at each location should be added to their location’s subgroup. Use bdr.join_node_group(), bdr.switch_node_group() or bdr.alter_node_group().
```

Comment:
Drop `alter_node_group` from this step.

Reply:
Which set? Joining the top-level group or adding the node to the local sub-group?

Reply:
Sorry, seems the marking is messed up. I meant the last on line 15: "Each node at each location should be added to their location’s subgroup."   There is no `bdr.alter_node_group` and variants of it that do exist don't allow you to add a node to a group. :-)